presto-docs/src/main/sphinx/functions/aggregate.rst

```diff
@@ -139,6 +139,22 @@ Approximate Aggregate Functions
 Statistical Aggregate Functions
 -------------------------------
 
+.. function:: corr(x, y) -> double
```

Comment:
All these arguments are in the wrong order. `y` should be the first argument. See `binary set function` in the grammer of the SQL spec